gui/src/components/form/SubmitButton.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+  import Button from '../Button.svelte';
+
+  // This is a right-aligned submit button for forms.
```

Comment:
aside: is the appropriate place for file comments? Seems like maybe they belong in a <!-- ... --> block above the script tag? or as // comments before the imports? no sure what the Svelte conventions are.

Reply:
Not sure, there's something off-putting about the `<!-- HTML style comments -->` and the way they are syntax highlighted plus it seems strange to not have imports at the top for me.

Happy to adopt whatever standard you'd like to establish here.